### PR TITLE
Maze runner

### PR DESCRIPTION
**Pull request by sanayamahajan-23**
## PR Description 📜

I added a new game as featurenamed Maze_Runner.It is a simple game in which the goal is to reac the endpoint.
Fixes # 4530

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

|[images](https://github.com/sanayamahajan-23/GameZone/blob/Maze_runner/Games/Maze_Runner/assets/images/Maze_Runner.png)

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @sanayamahajan-23 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: Games/Maze_Runner/assets/images/Maze_Runner.png
Comment (ishita-43) [inline Games/Maze_Runner/assets/images/Maze_Runner.png:1]: place this in the asset folder of the gamezone

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Also add your game entryinthe main readme.md file of the gamezone refer other entries for the format

File: Games/Maze_Runner/assets/images/Maze_Runner.png
Comment (sanayamahajan-23) [reply Games/Maze_Runner/assets/images/Maze_Runner.png:1]: done all the changes

**Comment by kunjgit** [comment]
Thank you  @sanayamahajan-23 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**